_docs/versioning.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Versioning your Docs
+description: How to turn on and use versioning
+---
+
+# Versioning
+
+Versioning documentation is a useful feature that can be achieved using Docsy Jekyll. It allows you to take snapshots of your documentation that may align with a specific version of a product or service, or you may wish to just keep a historical view of the changes that have been made to your documents over time.
+
+{% include alert.html type="info" title="Note: This only allows you to version documents inside the _docs folder, it does not allow you to version any files outside of that, such as Pages, Posts etc." %}
+
+
+## Implementation
+
+The way in which versioning is achieved is by duplicating your whole document directory and placing it inside an 'archive' subdirectory with the name of that version. The following illustrates what the base document root looks like *without* another version for this Docsy Jekyll site.
+
+```
+document root
+│      
+└─── _data
+└─── _docs
+        └─── extras
+        └─── subfolder
+        example-page.md
+        getting-started.md
+        subfolder.md
+        versioning.md
+```
+To create a version of this documentation we first create a folder with the version name we want to use inside of the 'archive' folder and copy all the files and folders from our root _docs folder into that new folder. That might look like this:
+
+<insert code block here - can't do alongside suggestion!>
```

Comment:
oh oops! So I had a code block in my suggested comment (and the only reason I couldn't include it is because it wouldn't fit within the suggestion) so I think we need to add it here. Sorry I wasn't clear!

The goal was to show the reader a quick way to do this copy.

Reply:
Here it is again:

```bash
# Our last release is version "Previous" so we are pinning the docs to this version`
mkdir -p _docs/Archive/Previous
# copy all content under _docs except Archive into this new folder.
cp -r _docs/!(Archive) _docs/Archive/Previous`
```

Reply:
Hmmm, odd as I added that to the instructions! Let me check and recommit if needs be...

Reply:
Somehow some files were missed, sorry about that. You should now have all the changes you mentioned.